services/vqa-service/docker/Dockerfile

```diff
@@ -0,0 +1,120 @@
+FROM opencog/vqa
```

Comment:
I don't think it makes sense to copy most of workshop docker infrastructure here. Did you want to make a copy of environment for testing purposes?

Reply:
Yes. I don't know yet where vqa service will be deployed.

Reply:
I think the better alternative will be a docker image which contains only VQA service itself and you can start it as standalone docker container which exports only gRPC port to interact with VQA service. Then you can use locally installed tools or another image (for instance workshop one) to publish your service to the Kovan or local network and test it. But you don't need to copy local network infrastructure to the VQA repo.

Reply:
That is good idea!